exchange/exchange_test.go

```diff
@@ -4175,37 +4175,29 @@ func TestMakeBidExtJSON(t *testing.T) {
 			description:        "Invalid extension, valid extBidPrebid and valid imp ext info",
 			ext:                json.RawMessage(`{invalid json}`),
 			extBidPrebid:       openrtb_ext.ExtBidPrebid{Type: openrtb_ext.BidType("video")},
-			impExtInfo:         map[string]ImpExtInfo{"test_imp_id": {true, []byte(`{"video":{"h":480,"mimes":["video/mp4"]}}`), json.RawMessage(`"prebid": {"passthrough": {"imp_passthrough_val": some_val}}"`)}},
 			expectedBidExt:     ``,
-			expectedErrMessage: "invalid character",
-		},
-		{
-			description:        "Valid extension, empty extBidPrebid and invalid imp ext info",
```

Comment:
Why did this test go away?

Reply:
This test was removed because the json-iterator `Marshal` function does not validate `json.RawMessage` fields like the standard library `Marshal` function so an error in `imp.ext` will go undetected. In that case I figured we should ensure that whatever is passed into this function is already known to be valid.

Reply:
Sounds good. These were attempts to test the error handling of Marshal, which may very well be a fools errand.